packages/connector-jsdom/src/connector.ts

```diff
@@ -421,7 +385,7 @@ export default class JSDOMConnector implements IConnector {
     }
 
     /* istanbul ignore next */
-    public querySelectorAll(selector: string): Promise<JSDOMAsyncHTMLElement[]> {
+    public querySelectorAll(selector: string): IHTMLElement[] {
```

Comment:
I think I mentioned it ealier but now that we use the same implementation everywhere we don't need an Interface, we can use directly a class or a type. Same goes for `IHTMLDocument` (looks like we also have an `HTMLDocument`)